opentracing/ext/tags.go

```diff
@@ -0,0 +1,49 @@
+package ext
+
+import (
+	"github.com/opentracing/api-golang/opentracing"
+)
+
+var (
+	// RPCTargetService records target service name for an RPC call
+	RPCTargetService = &stringTag{"rpc.target"}
```

Comment:
There is some intrinsic asymmetry (e.g., the "port" of the client is almost always ephemeral and rarely interesting, the client "service" is impossible to determine in some RPC systems), and – as you mention – some non-intrinsic asymmetry that nevertheless crops up due to implementation choices, like the ss/sr/cs/cr annotations.

But: I think we could still do well to try and consolidate... asymmetry or not, the fields have the same semantic meaning on client and server even if only a subset is available in either case. Howa bout calling them `"rpc.peer.*"` instead of `"rpc.*"`, though? Or even `"peer.*"` if you want to keep things more concise. Dapper newbies were sometimes confused by the fact that the peer was the one recording critical identity information.